packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -103,6 +109,35 @@ class MemoryController {
         .toList();
   }
 
+  void initializeLibraryFilters() {}
+
+  Future computeLibraries() async {
+    if (libraryCollection == null) {
+      final VM vm = await serviceManager.service.getVM();
+      // TODO(terry): Need to handle a possible Sentinel being returned.
+      final List<Isolate> isolates =
+          await Future.wait(vm.isolates.map((IsolateRef ref) async {
+        return await serviceManager.service.getIsolate(ref.id);
+      }));
+
+      libraryCollection = LibraryCollection(libraryFilters);
+      for (LibraryRef libraryRef in isolates[0].libraries) {
+        final Library theLibrary =
```

Comment:
make the requests to load the Library objects in parallel like you do above for the isolates case.

Reply:
Done.